polling_stations/templates/base.html

```diff
@@ -60,62 +60,33 @@
     {% block page_content %}
     {% endblock page_content %}
 {% endblock content %}
+{% block mailing_list %}{% endblock mailing_list %}
+{% block footer_menu %}
+    <div class="ds-cluster-center">
+        <p class="ds-h4">{% trans "Where Do I Vote?"%}</p>
+        <ul class="ds-padded" style="list-style:none;">
+            <li><a href="{% url 'home' %}">{% trans "Home" %}</a></li>
+            <li><a href="https://democracyclub.org.uk/projects/polling-stations/">{% trans "About Where Do I Vote?" %}</a></li>
+            <li><a href="https://democracyclub.org.uk/about/">{% trans "About Democracy Club" %}</a></li>
+            <li><a href="https://democracyclub.org.uk/privacy/">{% trans "Privacy" %}</a></li>
+            <li><a href="https://democracyclub.org.uk/contact">{% trans "Contact Us" %}</a></li>
+            <li><a href="https://democracyclub.org.uk/blog/">{% trans "Blog" %}</a></li>
+            <li><a href="https://twitter.com/democlub/">{% trans "Twitter" context "link to Democracy Club Twitter page" %}</a></li>
+            <li><a href="https://facebook.com/DemocracyClub/">{% trans "Facebook" context "link to Democracy Club Facebook page" %}</a></li>
+            <li><a href="https://github.com/DemocracyClub/">{% trans "GitHub" context "link to Democracy Club GitHub page" %}</a></li>
+        </ul>
+    </div>
+{% endblock footer_menu %}
+{% block extra_footer_text %}
+    <div class="ds-copyright">
+        <p>
+            {% blocktrans %}Contains OS data © Crown copyright and database right {{ current_year }}{% endblocktrans %}<br>
+            {% blocktrans %}Contains Royal Mail data © Royal Mail copyright and database right {{ current_year }}{% endblocktrans %}<br>
+            {% blocktrans %}Contains National Statistics data © Crown copyright and database right {{ current_year }}{% endblocktrans %}<br>
```

Comment:
Judging from your screen shot, I don't think `{{ current_year }}` is doing anything.

Reply:
I've pushed the missing variable `{% now "Y" as current_year %}` so works now. Screenshot updated, too.